src/web/app/pages-notifications/notifications-page.component.html

```diff
@@ -0,0 +1,43 @@
+<h1>Notifications</h1>
+<p>This page shows the active notifications visible to {{getUserTypeString()}}.</p>
+<div *ngIf="userType === NotificationTargetUser.INSTRUCTOR">
+  <p>To view notifications visible to students, please visit the <a href="/web/student/notifications">student notifications page</a>.</p>
+</div>
+
+<div class="row mb-3" *tmIsLoading="isLoadingNotifications">
+  <div class="col-12" *ngIf="notificationTabs.length > 0">
+    <div class="float-right">
+      <strong class="d-inline"> Sort By: </strong>
+      <div class="btn-group" data-toggle="buttons">
+        <button id="sort-end-time" class="btn btn-light" [disabled]="isSelectedForSorting(SortBy.NOTIFICATION_END_TIME)" (click)="sortNotificationsBy(SortBy.NOTIFICATION_END_TIME)">End Time (Ascending)</button>
```

Comment:
Is there a way to make this more intuitive I found the phrasing a bit confusing. Perhaps "ascending" be replaced with "latest first" and "descending" be "earliest first", though I can see how this would still cause confusion, any suggestions?

Reply:
To be consistent with other places (e.g., instructor home page), we can omit the ascending/descending altogether. Just mention the field used for sorting.

Reply:
Got it, I'll make the changes!